src/sbt-test/sbt-daffodil/tdml-saved-parser-01/test.script

```diff
@@ -0,0 +1,29 @@
+## Licensed to the Apache Software Foundation (ASF) under one
+## or more contributor license agreements.  See the NOTICE file
+## distributed with this work for additional information
+## regarding copyright ownership.  The ASF licenses this file
+## to you under the Apache License, Version 2.0 (the
+## "License"); you may not use this file except in compliance
+## with the License.  You may obtain a copy of the License at
+## 
+##  http://www.apache.org/licenses/LICENSE-2.0
+## 
+## Unless required by applicable law or agreed to in writing,
+## software distributed under the License is distributed on an
+## "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+## KIND, either express or implied.  See the License for the
+## specific language governing permissions and limitations
+## under the License.
+## 
+
+> packageDaffodilBin
+$ exists target/test-0.1-daffodil350.bin
+$ exists target/test-0.1-daffodil360.bin
+$ exists target/test-0.1-two-daffodil350.bin
+$ exists target/test-0.1-two-daffodil360.bin
+
+> Test/dependencyClasspath
+$ must-mirror target/tdmlparsers/test.bin target/test-0.1-daffodil360.bin
+$ must-mirror target/tdmlparsers/test-two.bin target/test-0.1-two-daffodil360.bin
+
+> test
```

Comment:
I was puzzled by this file, then I noticed daffodil-sbt has other test.script files, I looked at them, and I realized these files help test the daffodil-sbt plugin using a scripted test framework built into sbtPlugin itself.